tests/stbi_read_fuzzer.c

```diff
@@ -2,21 +2,31 @@
 extern "C" {
 #endif
 
+#include <stdint.h>
```

Comment:
This addition is here to make `stbi_read_fuzzer.c` compile on Windows: `stb_image.h` doesn't include `<stdint.h>` on Windows, but the [LLVMFuzzerTestOneInput](https://llvm.org/docs/LibFuzzer.html#id23) function signature traditionally uses the `uint8_t` and `size_t` types from `stdint.h`.